contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.2.0--4.3.0.sql

```diff
@@ -6783,12 +6784,11 @@ SELECT
 FROM pg_inherits pgi
 INNER JOIN pg_index cidx on cidx.indexrelid = pgi.inhrelid
 INNER JOIN pg_index pidx on pidx.indexrelid = pgi.inhparent
-INNER JOIN pg_class ctbl on ctbl.oid = cidx.indrelid
+INNER JOIN pg_class ctbl on (ctbl.oid = cidx.indrelid and ctbl.relkind = 'r' and ctbl.relispartition)
```

Comment:
I hope this will not leak any PG objects? If we do, did we can check with Alex if its fine since customers already might be using it as a workaround before this support for partitioning in TSQL?

Reply:
It will also list the partitions created it babelfish schemas from PG endpoint, I will check with Alex and block in separate PR depending on the reply.